src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/ExceptionHandler.kt

```diff
@@ -58,16 +41,10 @@ class ExceptionHandler {
 
 data class ErrorResponse(
   val status: Int,
-  val errorCode: Int? = null,
-  val userMessage: String? = null,
-  val developerMessage: String? = null,
-  val moreInfo: String? = null
+  val userMessage: String,
 ) {
   constructor(
     status: HttpStatus,
-    errorCode: Int? = null,
-    userMessage: String? = null,
-    developerMessage: String? = null,
-    moreInfo: String? = null
-  ) : this(status.value(), errorCode, userMessage, developerMessage, moreInfo)
+    userMessage: String = "An unexpected error has occurred with the JPC application, please contact support.",
```

Comment:
Stripped down to the bare minimum in terms of error information shown to the users.  The devs can view the logs to interrogate further.